praw/models/reddit/subreddit.py

```diff
@@ -733,9 +733,19 @@ def distinguish(self, thing, how='yes'):
             moderator level distinguish. 'no' removes any distinction. 'admin'
             and 'special' require special user priviliges to use.
 
+        :param sticky: Comment is stickied if True, placing it at the top of
+            the comment page regardless of score. If thing is not a top-level
+            comment, this parameter is silently ignored.
         """
+        data = {'how': how, 'id': thing.fullname}
+        if sticky:
+            thing_type = thing.fullname.split('_', 1)[0]
+            is_comment = (
+                thing_type == self.subreddit._reddit.config.kinds['comment']
```

Comment:
Is this the preferred way to check if `thing` is a comment? I took this pattern from `Comment.is_root`.